src/main/java/com/javadiscord/javabot/events/StarboardListener.java

```diff
@@ -27,326 +25,201 @@
 public class StarboardListener extends ListenerAdapter {
 
     void addToSB(Guild guild, MessageChannel channel, Message message) {
+        String guildId = guild.getId();
+        String channelId = channel.getId();
+        String messageId = channel.getId();
 
-        changeSBCBool(guild.getId(), channel.getId(), message.getId(), true);
-        TextChannel sc = guild.getTextChannelById(new Database().getConfigString(guild, "other.starboard.starboard_cid"));
+        Database db = new Database();
+        db.changeSBCBool(guildId, channelId, messageId, true);
+        TextChannel sc = guild.getTextChannelById(db.getConfigString(guild, "other.starboard.starboard_cid"));
 
-        var eb = new EmbedBuilder()
+        EmbedBuilder eb = new EmbedBuilder()
```

Comment:
Changing `var` to another variable type is something that doesn't really need to be changed. Either form is acceptable, so leave these as-is, otherwise it might turn into an edit war where people change it back and forth each time.